web/app/components/action.gts

```diff
@@ -7,7 +7,13 @@ interface ActionComponentSignature {
   };
 }
 
-export default class ActionComponent extends Component<ActionComponentSignature> {}
+export default class ActionComponent extends Component<ActionComponentSignature> {
+  <template>
+    <button type="button" class="action" ...attributes>
+      {{yield}}
+    </button>
+  </template>
+}
```

Comment:
The short-term downside of `.gts` is that GitHub doesn't have syntax highlighting yet.

Reply:
Looks like we should be able to configure this using an [override](https://github.com/github-linguist/linguist/blob/master/docs/overrides.md) in a `.gitattributes` file 🙌